### PR TITLE
chore: add CI checking for node ffi

### PR DESCRIPTION
**Pull request by Ma233**
### Please check if the PR fulfills these requirements

- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes / features)

## :large_blue_circle: What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
This PR added a simple smoke test for ffi feature.

## :information_source: Other information

Closes #issue

### CODE REVIEW

File: ffi/rings.py
Comment (RyanKung) [inline ffi/rings.py:1]: Perhaps organize it with a subpath such as 'examples/ffi' or 'node/tests/ffi/'.

File: .github/workflows/auto-release.yml
Comment (RyanKung) [inline .github/workflows/auto-release.yml:1]: We also need to make `rings.h` and `rings_node.dylib` downloadable.

File: node/build.rs [hunk 1]
```diff
@@ -14,7 +14,7 @@ fn gen_cbinding() {
         .generate()
     {
         Ok(g) => {
-            g.write_to_file("bindings.h");
+            g.write_to_file("../ffi/rings.h");
```
Comment (RyanKung) [inline node/build.rs:17]: It can be "OUT_DIR/include/rings.h"

File: .github/workflows/auto-release.yml
Comment (Ma233) [reply .github/workflows/auto-release.yml:1]: Should do in another PR.

File: node/build.rs (hunk 1 above)
Comment (Ma233) [reply node/build.rs]: "OUT_DIR" is not target directory. Updated to use "{CRATE_MANIFEST_DIR}/../target/include/rings.ht".

**Review by RyanKung — APPROVED** [review]

**Pull request merged**